bundles/statistics/statsgrid2016/components/IndicatorParametersHandler.js

```diff
@@ -25,10 +29,11 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.IndicatorParameterHandler', fun
         var errorService = me.service.getErrorService();
         var locale = this.locale;
         var panelLoc = locale.panels.newSearch;
+        var elements = me.elements || {};
 
         this.service.getIndicatorMetadata(this.datasource, indId, function (err, indicator) {
-            if (me.elements.dataLabelWithTooltips) {
-                me.elements.dataLabelWithTooltips.find('.tooltip').hide();
+            if (elements.dataLabelWithTooltips) {
+                elements.dataLabelWithTooltips.find('.tooltip').hide();
```

Comment:
What are these "elements.dataLabelWithTooltips"? Some elements that's being passed around between classes? Feels like they shouldn't be referenced in so many places.